core/src/test/kotlin/org/stellar/anchor/sep31/Sep31ServiceTest.kt

```diff
@@ -326,11 +351,84 @@ class Sep31ServiceTest {
     // TODO: Add fee validation.
     assertEquals("100", txn.amountIn)
     assertEquals(
-      "stellar:USDC:GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN",
-      txn.amountInAsset,
+        "stellar:USDC:GA5ZSEJYB37JRC5AVCIA5MOP4RHTM335X2KGX3IHOJAPP5RE34K4KZVN",
+        txn.amountInAsset,
     )
     assertEquals("12500", txn.amountOut)
     assertEquals(stellarJPYC, txn.amountOutAsset)
+
+    // Test null quote failure
+    Context.get().setQuote(null)
+    assertThrows<ServerErrorException> { sep31Service.updateTxAmountsBasedOnQuote() }
```

Comment:
💡  Can you assert the error class and message here? We do it in some places like:
https://github.com/stellar/java-stellar-anchor-sdk/blob/ece30f35d6fe37dfee2dab6df33791fd0d3ec470/core/src/test/kotlin/org/stellar/anchor/sep12/Sep12ServiceTest.kt#L49-L53

Reply:
The type can actuallybe done by Java generics like assertThrows<ServerErrorException>.

I had added the message check.